docs/doc_v1.0/Build_documentation.md

```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
-to the mix. This is done by running a random number test against each
-susceptible case and setting their status to EXPOSED.
+![](RunContactData.png)
 
-Following this, the contact data for this timestep is extracted along
-with the number of active cases (those not S, R or D), if this is zero
-and random infections are off, it will exit here. Otherwise, it will
-loop over the contact data.
+### Contact Event
+
+Contact events are set in the contacts.csv file. They involve the interaction between two people at a given time. 
+The duration/proximity of the event is measured using the weight parameter. 
+
+When a contact event takes place the two people are assessed based on their Virus Statuses, this allows the model to 
+assess if the contact has the potential to spread the infection. One or both of the contacts may be in isolation, 
+so the contact may not take place. The isolation policies are expanded upon below. 
+
+![](ContactEvent.png)
+
+### Alert Event
+
+The alert event processor is used to cycle through the states of disease alert that a member of the population can be in at any one point. 
+
+![](image6.png)
 
-The contact data is evaluated using a series of conditionals. Firstly,
-the alert status and weight of the contact are reviewed to see if this
-is a significant contact. If both alert statuses are NONE and the weight
-of the contact is greater than exposureThreshold, it is a contact of
-interest. Before evaluating the contact further, the two cases are
-checked to see if their virus statuses are different, if they are the
-contact is evaluated.
+The person has their status set to a corresponding status and the next status is calculated for a time in the future. 
+The result of the test is determined by whether the individual is currently infectious and a dice roll against the accuracy 
+of the test. The number of false positives and negatives is recorded as an output statistic. 
 
-The two cases are evaluated to see who is in the more severe state,
-these are described by the rankings:
+![](AlertEvent.png)
 
-| \-1 | DEAD           |
-| :-- | :------------- |
-| 0   | SUSCEPTIBLE    |
-| 1   | RECOVERED      |
-| 2   | EXPOSED        |
-| 3   | EXPOSED\_2     |
-| 4   | INFECTED       |
-| 5   | INFECTED\_SYMP |
+### Infection Event
 
-PersonA is the more severe case; the least severe case is personB. A
-dangerous mix is when personA is infectious (E2, I, IA) and personB is
-susceptible. If this is the case, the following test is conducted:
+An infection event is triggered in three ways:
 
-![](eq6.gif)
+1. The id has been chosen as a seed infection.
+2. The id has been randomly chosen to be a seed.
+3. The id has been in contact with an infectious individual and has subsequently contracted the infection. 
 
-Where n is a random number.
+Once a SUSCEPTIBLE individual is exposed to the infection, an infection event is triggered which in turn creates a virus event. 
```

Comment:
Is this still true given the order of the event processing in EventRunnerImpl.java has been reversed?